scripts/normalize-by-median.py

```diff
@@ -243,6 +243,10 @@ def get_parser():
 
 
 def main():  # pylint: disable=too-many-branches,too-many-statements
+    """
```

Comment:
I'd disable docstrings for main functions of scripts. There is no information here.